diffrax/_solver/srk.py

```diff
@@ -198,8 +202,9 @@ class AbstractSRK(AbstractSolver[_SolverState]):
     r"""A general Stochastic Runge-Kutta method.
 
     This accepts `terms` of the form
-    `MultiTerm(ODETerm(drift), ControlTerm(diffusion, brownian_motion))`.
-     Depending on the solver, the Brownian motion might need to generate
+    `MultiTerm(ODETerm(drift), ControlTerm(diffusion, brownian_motion))` or
+    `MultiTerm(ODETerm(drift), WeaklyDiagonalControlTerm(diffusion, brownian_motion))`.
```

Comment:
I think it is in fact more general and accepts anything of the form `MultiTerm[tuple[ODETerm, AbstractTerm[Any, self.minimal_levy_area]]]`. This will be relevant for underdamped Langevin where a custom diffusion term is used, because the BM term only appears in dv, not in dx (and a WeaklyDiagonalTerm would be inefficient). But I agree if you want to keep it as you wrote it here, to avoid confusing users.

-- Nevermind, I see you adressed this later in `sde_solvers.md`.